app/components/Heads.js

```diff
@@ -0,0 +1,24 @@
+const React = require('react');
+const { defineSvg } = require('../defineSvg');
+
+module.exports = defineSvg('Heads', {
+  className: 'tilt-icon-heads',
+
+  render() {
+    return (
+      <svg className={this.getClassNames()} viewBox="0 0 20 17">
+        <title>Heads</title>
+        <g className="tilt-icon-main">
```

Comment:
same as above, consolidate all the `transform=translate` into a single `<g/>` (or remove if not needed)